crates/router/src/services/authentication/user_authentication_method.rs

```diff
@@ -0,0 +1,17 @@
+use common_enums::{Owner, UserAuthType};
+use diesel_models::UserAuthenticationMethod;
+use once_cell::sync::Lazy;
+
+pub static DEFAULT_USER_AUTH_METHOD: Lazy<UserAuthenticationMethod> =
+    Lazy::new(|| UserAuthenticationMethod {
+        id: String::from("hyperswitch_default"),
+        auth_id: String::from("hyperswitch"),
+        owner_id: String::from("hyperswitch"),
+        owner_type: Owner::Tenant,
+        auth_type: UserAuthType::Password,
+        private_config: None,
+        public_config: None,
+        allow_signup: true,
+        created_at: common_utils::date_time::now(),
+        last_modified_at: common_utils::date_time::now(),
+    });
```

Comment:
Should we change the placement for this file @racnan?

Reply:
yes, this is different `user_authentication_methods` is not a part of  `authentication`. Sounds similar but are not.
`authentication` modules deals with api authentication
`user_authentication_methods` deals with control centre user authentication (logging in).

This should be in some user util.

Reply:
putting it in `domain/user`